package.json

```diff
@@ -62,11 +61,16 @@
     "@emotion/css": "11.10.6",
     "@grafana/data": "10.1.0",
     "@grafana/runtime": "10.1.0",
-    "@grafana/ui": "10.1.0",
     "@grafana/schema": "10.1.0",
+    "@grafana/ui": "10.1.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "tslib": "2.5.3"
   },
+  "resolutions": {
+    "jest/**/string-width": "4.2.3",
+    "isaacs/**/wrap-ansi": "8.1.0",
+    "jest/**/wrap-ansi": "7.0.0"
+  },
```

Comment:
why we need this hardcoded resolutions ? This should have comment at least in git commit message, so in future we can remove it, once we understand why it is there and we can evaluate if it is still needed

Reply:
After installing playwright I wasn't able to run the jest tests anymore. When doing yarn install I got the warning 
```
warning Pattern ["string-width@^4.1.0"] is trying to unpack in the same destination ".../Library/Caches/Yarn/v6/npm-string-width-cjs-4.2.3-269c7117d27b05ad2e536830a8ec895ef9c6d010-integrity/node_modules/string-width-cjs" as pattern ["string-width-cjs@npm:string-width@^4.2.0"]. This could result in non-deterministic behavior, skipping.
```
and when running the jest tests I got the following error

```
Error [ERR_REQUIRE_ESM]: require() of ES Module /home/runner/work/analytics-grafana-datasource/analytics-grafana-datasource/node_modules/string-width/index.js from /home/runner/work/analytics-grafana-datasource/analytics-grafana-datasource/node_modules/cliui/build/index.cjs not supported.
Instead change the require of index.js in /home/runner/work/analytics-grafana-datasource/analytics-grafana-datasource/node_modules/cliui/build/index.cjs to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/home/runner/work/analytics-grafana-datasource/analytics-grafana-datasource/node_modules/cliui/build/index.cjs:2[9](https://github.com/bitmovin/analytics-grafana-datasource/actions/runs/10098430816/job/27925459055#step:8:10)1:21)`
```
This is a bug in the cache system of yarn v1. I found two suggested ways to solve this:
1) update yarn to latest version. I tried this, and although the resolution then worked correctly, I didn't manage to get the pregenerated eslint configuration from grafana to work with the updated yarn. OR
2) add those resolutions as suggested [here](https://github.com/cypress-io/cypress/issues/27370) and [here](https://github.com/storybookjs/storybook/issues/22431)